Package.swift

```diff
@@ -0,0 +1,45 @@
+// swift-tools-version:5.3
+
+import PackageDescription
+let package = Package(
+    name: "Checkout3DSPackages",
+    platforms: [
+        .iOS(.v12)
+    ],
+    products: [
+        .library(
+            name: "Checkout3DSPackages",
+            targets: ["Checkout3DSPackages"]
+        )
+    ],
+    dependencies: [
+        .package(
+            url: "https://github.com/checkout/checkout-event-logger-ios-framework.git",
+            from: "1.2.4"
```

Comment:
Test : What happens with different version of checkout-event-logger-ios-framework with Checkout3DSPackages and Frames

Reply:
So what happens if Frames is not compatible with 1.2.4?. If Checkout3DSPackages is configured to use the latest Does it force Frames use the latest version?